float64 yaw
float64 pitch
float64 roll
---
float64 w
float64 x
float64 y
float64 z